workos/__about__.py

```diff
@@ -12,7 +12,7 @@
 
 __package_url__ = "https://github.com/workos-inc/workos-python"
 
-__version__ = "1.10.0"
+__version__ = "1.11.0.a1"
```

Comment:
Synced with the eng team today. We're going to be publishing MFA changes to the stable channels so we don't need to worry about prerelease suffixes.
```suggestion
__version__ = "1.10.0"
```